notebooks/Taxi_Dropoff_Reverse_Geocoding.ipynb

```diff
@@ -0,0 +1,504 @@
+{
```

Comment:
<p>I would suggest using the less colloquial title here. So perhaps "Construct human readable street addresses from NAD database" and "Gather rows based on the indices of the nearest address"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1001/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>